verl/trainer/config/ppo_megatron_trainer.yaml

```diff
@@ -229,7 +229,8 @@ reward_model:
   use_dynamic_bsz: ${critic.use_dynamic_bsz}
   max_length: null
   launch_reward_fn_async: False # custom reward function executed async on CPU, during log_prob
-
```

Comment:
let's make it hierarchical:
sandbox_fusion:
   url:
   xxxx:

Reply:
Pushed a new commit to fix this. Please review again.